pdns/README-dnsdist.md

```diff
@@ -214,16 +214,16 @@ The supported values for `source` are:
 
  * an IPv4 or IPv6 address, which must exist on the system
  * an interface name
- * an IPv4 or IPv6 address followed by '@' then an interface name
+ * an IPv4 or IPv6 address followed by `@' then an interface name
```

Comment:
Oops, I changed that to `` but it became ugly and then I only half changed it back.